src/mlpack/tests/cf_test.cpp

```diff
@@ -160,10 +160,8 @@ void GetRecommendationsQueriedUser()
  */
 template<typename DecompositionPolicy,
          typename NormalizationType = NoNormalization>
-void RecommendationAccuracy(const size_t allowedFailures = 17)
+void RecommendationAccuracy(size_t allowedFailures = 17, DecompositionPolicy& decomposition = DecompositionPolicy())
```

Comment:
@zoq, this gives the following error:

error: non-const lvalue reference to type 'mlpack::cf::RandomizedSVDPolicy' cannot bind to a temporary of type 'mlpack::cf::RandomizedSVDPolicy'
void RecommendationAccuracy(size_t allowedFailures = 17, DecompositionPolicy& decomposition = DecompositionPolicy())
                                                                              ^               ~~~~~~~~~~~~~~~~~~~~~
/mlpack/src/mlpack/tests/cf_test.cpp:668:3: note: in instantiation of default function argument expression for 'RecommendationAccuracy<mlpack::cf::RandomizedSVDPolicy, mlpack::cf::NoNormalization>' required here
  RecommendationAccuracy<RandomizedSVDPolicy>();
  ^
/mlpack/src/mlpack/tests/cf_test.cpp:163:79: note: passing argument to parameter 'decomposition' here
void RecommendationAccuracy(size_t allowedFailures = 17, DecompositionPolicy& decomposition = DecompositionPolicy())

Reply:
Ahh, either remove the reference (`DecompositionPolicy decomposition`) or use `const`, so `const DecompositionPolicy& decomposition`.